data/companies.yml

```diff
@@ -580,3 +580,6 @@ users:
   - name: "GoTo Financial"
     logo: "/logos/gotofinancial.svg"
     url: "https://gotofinancial.com/"
+  - name: "GoHighLevel"
```

Comment:
Also, place this alphabetically in the list.

Reply:
Sure @ericvn